Zone5Tests/Transport/URLRequestInterceptorTests.swift

```diff
@@ -91,42 +88,57 @@ class URLRequestInterceptorTests: XCTestCase {
 		XCTAssertNil(interceptor.lastRequest)
 		XCTAssertEqual(request, interceptor.request)
 		
+		let tokenNotificationExpectation = self.expectation(description: "notification for token fired")
```

Comment:
I'd suggest using an `XCTNSNotificationExpectation` for testing notifications.

Reply:
ok, I'll check this out.

Reply:
done